rigex-front-end/src/components/vital-resources/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import './headerStyle.scss'
+import building from '../../assets/building2.jpeg' 
```

Comment:
Let's use faker for all our images. You should not need any asset files for this sprint